MasumaJaffery/Catalog_of_My_Things
# Implement SQL Schema for Genres and Music Albums Tables

This Pull Request introduces the SQL schema for the genres and music_albums tables, as part of the ongoing effort to build the catalog_of_my_things database. These tables are structured to store details related to music albums and their genres, incorporating all relevant properties and associations from the parent Item class as table columns.

**Changes Included:**

- Created the genres table with id and name columns.
- Created the music_albums table with columns including id, title, publish_date, archived, on_spotify, last_played_at, genre_id, source_id, label_id, and author_id.
- Established foreign key relationships in the music_albums table, referencing genres, sources, labels, and Author tables.

**Notes:**

- The music_albums table assumes the existence of sources and labels tables.
- Proper testing should be conducted to ensure the integrity and functionality of the created tables within the database environment.

**How to Test:**

- Run the SQL script in your local database environment.
- Verify the successful creation of the genres and music_albums tables with the correct columns and data types.
- Check the established foreign key relationships and constraints.
- Optionally, insert some test data to ensure data integrity and relationships are maintained.

Please review the introduced schema and let me know if any adjustments are needed.

Closes #13
